SpringProject/src/main/java/com/gradproject2019/conferences/controller/ConferenceController.java

```diff
@@ -14,8 +15,15 @@
     private ConferenceService conferenceService;
 
     @GetMapping(path = "/conferences")
-    public @ResponseBody
-    Iterable<Conference> getAllUsers() {
+    @ResponseBody
+    public Iterable<Conference> getAllUsers() {
         return conferenceService.listConferences();
     }
+
+    @GetMapping(path = "/conferences/{id}")
+    @ResponseBody
+    public Conference getConferenceById(@PathVariable(required = false) Long conferenceId) {
```

Comment:
@PathVariable(required = false), usually path variables are mandatory. If you need them to be optional it's better to use RequestParams